turbinia/lib/libcloudforensics.py

```diff
@@ -554,6 +614,109 @@ def get_source_string(self):
     """
     return self.get_value('selfLink')
 
+  def get_resource_type(self):
+    """Get the resource type from the resource key-value store.
+
+    Returns:
+      Resource Type which is a string with one of the following values:
+      compute#instance
+      compute#disk
+      compute#snapshot
+    """
+    if not self._data:
+      operation = self.get_operation().execute()
+      self._data = self.project.gce_operation(
+          operation, zone=self.zone, block=False)
+    return self._data['kind']
+
+  def form_operation(self, operation_name):
+    """Form an API operation object for the compute resource.
+
+    Example:[RESOURCE].form_operation('setLabels')(*args)
```

Comment:
I think you mean `**kwargs` here

Reply:
yes will change this, it takes only key-worded parameters